cmd/tracee-ebpf/printer.go

```diff
@@ -91,15 +91,15 @@ func (p tableEventPrinter) Init() error { return nil }
 func (p tableEventPrinter) Preamble() {
 	if p.verbose {
 		if p.containerMode {
-			fmt.Fprintf(p.out, "%-16s %-16s %-13s %-12s %-12s %-6s %-16s %-15s %-15s %-15s %-16s %-20s %s", "TIME", "UTS_NAME", "CONTAINER_ID", "MNT_NS", "PID_NS", "UID", "COMM", "PID/host", "TID/host", "PPID/host", "RET", "EVENT", "ARGS")
+			fmt.Fprintf(p.out, "%-16s %-16s %-13s %-12s %-12s %-6s %-16s %-15s %-15s %-15s %-16s %-22s %s", "TIME", "UTS_NAME", "CONTAINER_ID", "MNT_NS", "PID_NS", "UID", "COMM", "PID/host", "TID/host", "PPID/host", "RET", "EVENT", "ARGS")
```

Comment:
what are these?

Reply:
The event name is too long so we need to increase the max event name in the table printer

Reply:
The table is already too wide. I think it is better to have a shorter name rather than making this field size bigger

Reply:
Wait, but I see you already made the name shorter (which is fine by me), didn't you?

Reply:
Yes that's an old comment that was pending - you can ignore it